src/pillarStyles.ts

```diff
@@ -60,10 +60,18 @@ export const pillarColours: PillarColours = {
         inverted: palette.lifestyle[500],
         liveblogBackground: palette.lifestyle[300],
         liveblogDarkBackground: palette.lifestyle[200]
+    },
+    [Special.SpecialReport]: {
+        kicker: palette.lifestyle[400],
+        featureHeadline: palette.lifestyle[300],
+        soft: palette.lifestyle[800],
+        inverted: palette.lifestyle[500],
+        liveblogBackground: palette.lifestyle[300],
+        liveblogDarkBackground: palette.lifestyle[200]
     }
 }
 
-const getPillarStyles = (pillar: Pillar): PillarStyles => pillarColours[pillar];
+const getPillarStyles = (pillar: Theme): PillarStyles => pillarColours[pillar];
 
 function pillarFromString(pillar: string | undefined): Pillar {
     switch (pillar) {
```

Comment:
Some of the names in this file might be a bit confusing now? For example, `PillarColours` where the key is `theme`?